stable/atlantis/Chart.yaml

```diff
@@ -1,17 +1,15 @@
 apiVersion: v1
+# The prometheus-operator chart is deprecated and no longer maintained. For details deprecation,
```

Comment:
@torstenwalter looks like you're setting "prometheus-operator" comments for charts that aren't the prometheus-operator.

Reply:
That was a copy paste mistake. Will create a PR to fix it.

Reply:
As promised https://github.com/helm/charts/pull/23855